src/index.tsx

```diff
@@ -1,3 +1,4 @@
+import '@/wdyr'
```

Comment:
why not `./wdyr` instead

Reply:
lint issue, this must be on top of the file. but prettier order import will move it to the bottom